plugins/org.eclipse.fordiac.ide.typemanagement/src/org/eclipse/fordiac/ide/typemanagement/refactoring/ReplaceVarsWithStructChange.java

```diff
@@ -0,0 +1,57 @@
+package org.eclipse.fordiac.ide.typemanagement.refactoring;
```

Comment:
can you make  a new package for classes ?

org.eclipse.fordiac.ide.typemanagement.refactoring.connection
So we put all the connection related refactorings there, and the general changes, or commands, which might be reused by several refacotrings should be part of the top level org.eclipse.fordiac.ide.typemanagement.refactoring package